tf_files/aws/modules/sqs/cloud.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_sqs_queue" "my_queue" {
```

Comment:
Nit-picking, but is there a more meaningful name than `my_queue`? `generic_queue`?